cmake/project-config.cmake.in

```diff
@@ -12,9 +12,25 @@ if(@PROJECT_VARIANT_NAME@ STREQUAL "PROJ4")
 endif()
 
 include(CMakeFindDependencyMacro)
+
+cmake_policy(PUSH)
+cmake_policy(SET CMP0012 NEW)
+if("@ENABLE_TIFF@")
+    find_dependency(TIFF)
+endif()
+cmake_policy(POP)
+
 if("@CURL_ENABLED@" STREQUAL "TRUE")
```

Comment:
With the scope of `CMP0012` extended to this block, this can condition be simplified to `if("@CURL_ENABLED@")`, for consistency.

Reply:
good point. edited